xp/.github/workflows/test.yml

```diff
@@ -0,0 +1,45 @@
+name: CI
```

Comment:
i believe this has to go in the root directory. eg moondao/.github/workflows for GitHub to find it.

also you could merge just this file into main so that XP ci will run on this PR